src/api/app/views/webui2/webui/kiwi/images/_build_info.html.haml

```diff
@@ -0,0 +1,39 @@
+%ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible
+  %li.nav-item
+    %a.nav-link.active.result-select-link#result-select-link{ href: '#' } Build Results
+
+.result-display
+.card-body
+  .btn.btn-sm.btn-outline-primary.build-refresh{ onclick: 'update_build_result()', title: 'Refresh Build Results' }
```

Comment:
We try to split javascript code from the html views, often called [unobtrusive javascript](https://guides.rubyonrails.org/working_with_javascript_in_rails.html#unobtrusive-javascript). Please move this to a js file or into the js part of this file.